core/templates/pages/contributor-dashboard-page/translation-opportunities/translation-opportunities.component.ts

```diff
@@ -123,6 +125,13 @@ angular.module('oppia').component('translationOpportunities', {
             getPresentableOpportunitiesData);
       };
 
+      ctrl.resetCursorStyle = function() {
+        document.body.style.cursor = '';
+        // Function used to reset cursor style to normal if copy mode is left on
+        // and the translation modal is closed down.
```

Comment:
Probably makes more sense to move the comment to where the function is being called (lines 103 and 105), to explain why the cursor needs to be reset there (the function itself is pretty self-explanatory).

Reply:
Yep, that's true.